codespell_lib/data/dictionary.txt

```diff
@@ -233,6 +233,7 @@ accurancy->accuracy
 accurarcy->accuracy
 accurary->accuracy, actuary,
 accuray->accuracy, actuary,
+accurs->occurs
```

Comment:
This is a legit word per https://en.oxforddictionaries.com/definition/accurse  
What do think @larsoner ?

Reply:
I may be misunderstanding your comment, but the link refers to `accurse` (with the last letter `e`) but the typo is for `accurs` (with last letter `s`).

`accurse` *is* a legit word but `accurs` is not.

Or did you mean that `accurse` could be a possible correction also?

While we are talking about code which might well be accursed, I think "occurs" would be more likely. :)

Reply:
Added